packages/material-ui/src/OverridableComponent.d.ts

```diff
@@ -3,7 +3,7 @@ import { Omit } from '@material-ui/types';
 import { StyledComponentProps } from './styles';
 
 /**
- * a component whose root component can be controled via a `component` prop
+ * a component whose root component can be controlled via a `component` prop
```

Comment:
Nice spot, what do you think about adding
```suggestion
 * A component whose root component can be controlled via a `component` prop.
```

Reply:
Looking at the rest of the file, the same can be applied to the other comments.

Reply:
@joshwooding 
What exactly can be applied?

Reply:
Making the comments begin with a capital and adding a full stop at the end.